src/session/SessionManager.h

```diff
@@ -163,19 +179,18 @@ class NetSessions {
 	// the new one.  Connection count stats get updated either way (so most
 	// cases should likely check that the key is not already in the map to
 	// avoid unnecessary incrementing of connecting counts).
-	void InsertConnection(ConnectionMap* m, const detail::ConnIDKey& key, Connection* conn);
+	void InsertSession(detail::SessionKey key, Session* session);
 
-	ConnectionMap tcp_conns;
-	ConnectionMap udp_conns;
-	ConnectionMap icmp_conns;
-
-	SessionStats stats;
-
-	analyzer::stepping_stone::SteppingStoneManager* stp_manager;
-	detail::PacketFilter* packet_filter;
+	SessionMap session_map;
+	detail::ProtocolStats* stats;
 };
 
 // Manager for the currently active sessions.
-extern NetSessions* sessions;
+extern SessionManager* session_mgr;
```

Comment:
For other managers, we have them inside the global `zeek::*` namespace, so suggest we do the same here.